src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVMBackup.md

```diff
@@ -22,7 +22,7 @@ Remove-AzureRmVMBackup [-ResourceGroupName] <String> [-VMName] <String> [-Tag] <
 
 ### 1:
 ```
-PS C:\>
```

Comment:
This looks like no example whatsover, why aren't we removing the example, or better yet, adding an example?